app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<section class="spotlight">
+<% media = Work.all.sample%>
+    <h2 class="spotlight__header">
+        <span class="spotlight__header--prefix"> Media Spotlight </span>
+        <a class="spotlight__description"><%= link_to media.title, work_path(media.id)%> by <%= media.creator%></a>
+    </h2>
+    <p class="spotlight__description"><%= media.description%></p>
+</section>
+
+<section class="top-ten__container">
+  <section class="top-ten__list-container"> 
+    <h2>Top Movies</h2>
+      <%= render "top_ten", top: @top_ten[2] %>
+  </section>
```

Comment:
I love the way you've used a partial to DRY this up - it makes this code much more readable.